clients/fides-js/src/lib/shared-consent-utils.ts

```diff
@@ -48,3 +48,20 @@ export const transformConsentToFidesUserPreference = (
   }
   return UserConsentPreference.OPT_OUT;
 };
+
+/**
+ * Parses a comma-separated string of notice keys into an array of strings.
+ * Handles undefined input, trims whitespace, and filters out empty strings.
+ */
+export const parseFidesDisabledNotices = (
+  value: string | undefined,
+): string[] => {
+  if (!value) {
+    return [];
+  }
+
+  return value
+    .split(",")
+    .map((key) => key.trim())
+    .filter(Boolean);
+};
```

Comment:
I moved this here from `consent-utils.ts` to resolve a circular dependency warning in Rollup.